web/logging.py

```diff
@@ -17,8 +18,11 @@ def __init__(
         super().__init__(app)
 
     async def dispatch(self, request: Request, call_next: Callable) -> Response:
-
-        request_id: str = str(uuid4())
+        transaction = sentry_sdk.get_current_scope().transaction
+        if transaction is not None:  # pragma: no cover
+            request_id = transaction.trace_id

```

Comment:
Just a teeny tiny comment here,  why is `# pragma: no cover` required here? Why is it intentionally left out of coverage?

Reply:
Couple of things actually:
1. Low priority as this part of the code runs well whether Sentry is configured or not.
2. The gains of writing the test for this is not really worth the effort.

We can definitely write the test for it in another task/PR but for now I believe it's inconsequential, hence the exclusion

Reply:
Based. I can relate to your rationale. You have my sympathy. APPROVED!